source/retryable-writes/tests/README.md

```diff
@@ -71,7 +71,7 @@ Drivers should also assert that command documents are properly constructed with
 on whether the write operation is supported.
 [Command Logging and Monitoring](../../command-logging-and-monitoring/command-logging-and-monitoring.rst) may be used to
 check for the presence of a `txnNumber` field in the command document. Note that command documents may always include an
-`lsid` field per the [Driver Session](../../sessions/driver-sessions.md) specification.
+`lsid` field per the [Driver Session](../../sessions/driver-sessions.rst) specification.
```

Comment:
The session spec was converted in https://github.com/mongodb/specifications/commit/c6349d199588fa6cd9ea3e304366f90268ea3091. Shouldn't this remain as `.md`?

Reply:
Fixed